test/Microsoft.IdentityModel.Tests/ReferenceXml.cs

```diff
@@ -561,6 +562,215 @@ public static KeyInfoTestSet KeyInfoMalformedCertificate
         }
         #endregion
 
+        #region Saml
+
+        #region SamlAction
+        public static SamlActionTestSet SamlActionValueNull
+        {
+            get
+            {
+                return new SamlActionTestSet
+                {
+                    Action = Default.SamlAction,
+                    Xml = XmlGenerator.SamlActionXml(SamlConstants.Namespace, Default.SamlAction.Namespace.ToString(), null)
+                };
+            }
+        }
+
+        public static SamlActionTestSet SamlActionValueEmpthString
+        {
+            get
+            {
+                return new SamlActionTestSet
+                {
+                    Action = Default.SamlAction,
+                    Xml = XmlGenerator.SamlActionXml(SamlConstants.Namespace, Default.SamlAction.Namespace.ToString(), String.Empty)
+                };
+            }
+        }
+
+        public static SamlActionTestSet SamlActionNoNamespace
```

Comment:
To be consistent, perhaps use the name SamlActionNamespaceNull {similar to SamlActionValueNull}